tests/meson.build

```diff
@@ -158,6 +158,7 @@ tests = {
   'genkey': {'suites': all_suites},
   'pkey': {'suites': all_suites},
   'session': {'suites': all_suites},
+  'skey': {'suites': ['kryoptic', 'kryoptic.nss']},
```

Comment:
the softhsm/sofotkn do not work here? It would be good to add a notes here when not all suites are supported why it is.

Reply:
ok softhsm deadlocks in openssl enc
and softoken fails key generation with CKR_USER_TYPE_INVALID ...

Reply:
The sotfokn failure sounds weird and I think would deserve some investigation. But it can be done later.